source/stylesheets/refills/unstyled/_navigation.scss

```diff
@@ -0,0 +1,82 @@
+.navigation {
+  $large-screen: em(860) !default;
+  $large-screen: $large-screen;
+
+  // Mobile view
+  .navigation-menu-button {
+    display: block;
+    float: right;
+    margin: 0;
+    padding-top: 0.5em;
+
+    @include media ($large-screen) {
+      display: none;
+    }
+  }
+
+  // Nav menu
+  .navigation-wrapper {
+    @include clearfix;
+    position: relative;
+  }
+
+  .logo {
+    float: left;
+  }
+
+  .logo img {
```

Comment:
Merge rule `.logo img` with rule on line 23<br>Selector should have depth of applicability no greater than 2, but was 3